onsite-groups.html

```diff
@@ -13,7 +14,7 @@ <h1>
     </svg>
   </h1>
   <p>
-    It’s true. We get by with a little help from our friends. Find yours here.
+    Itâ€™s true. We get by with a little help from our friends. Find yours here.
```

Comment:
Nope.

Reply:
That's really weird, I didn't change that. 🤔 

Pushing up a commit to change it back.